source/developers/how-tos/get-ready-for-python-dev.rst

```diff
@@ -0,0 +1,82 @@
+How To Get Ready for Python Development
+#######################################
+
+.. How-tos should have a short introduction sentence that captures the user's goal and introduces the steps.
+
+This how-to will help prepare your local system to start developing on a Python-based service that is part of the Open edX Platform.
+
+Assumptions
+***********
+
+.. This section should contain a bulleted list of assumptions you have of the
+   person who is following the How-to.  The assumptions may link to other
+   how-tos if possible.
+
+* You have Git installed
+* You have Python 3.8 installed
+
+Steps
+*****
+
+.. note::
+
+   Some repositories will have a variation of these instructions in their README. Often it will be better to follow the instructions here instead, as the README instructions may be outdated, but keep an eye out for repositories that document special workflows.
+
+First-Time Setup
+================
+
+First, clone the repository with ``git clone <git_url>``. It's best to place all of your Open edX repositories in the same parent directory as each other, as a few of them assume that they will be arranged this way. (If you're not using devstack, this may not matter.)
+
+After navigating to the new directory in your shell, you'll need to set up a virtualenv and install requirements.
+
+A virtualenv contains a copy of your repo's Python dependencies. This avoids dependency version conflicts with your system packages and other repos.
+
+The most basic way of creating a virtualenv is to use the builtin ``venv`` module in Python::
+
+  python3.8 -m venv .venv-3.8
```

Comment:
I kind of understand why the `-3.8` suffix is there, but these instructions will only involve one venv, so maybe we can keep it simpler by skipping the suffix.